### ISSUE TITLE
This Theme Makes Atom Slow

### ISSUE THREAD

**Issue by cuiweiqiang** [silvestreh/atom-material-ui#116]
when I use this theme, like this:
![image](https://cloud.githubusercontent.com/assets/863696/11110083/a6ec01c8-8932-11e5-9f7b-2d2834120357.png)
it makes atom slow when switch tabs.
Is there any solutions ?

**Comment by mackelito** [comment]
I have no problems with that... perhaps there is another plugin that when used in this theme can cause this issue?

**Comment by monkpit** [comment]
I have this issue as well. Running Atom on Windows 7.

**Comment by mackelito** [comment]
Ah.. I'm running El Capitan...

**Comment by silvestreh** [comment]
I have no performance issues on a early 2011 13" macbook pro running yosemite. This is, by no means, a fast machine by today's standards.

I'll check how it performs on a Windows machine as soon as I get my hands on one.

**Comment by cuiweiqiang** [comment]
This is my community packages list:
![image](https://cloud.githubusercontent.com/assets/863696/11138115/32fce904-89f9-11e5-993c-0f98887a5d9a.png)
And this is my community themes list:
![image](https://cloud.githubusercontent.com/assets/863696/11138122/52f93ba4-89f9-11e5-8935-d49f5371fe06.png)

I'm running El Capitan too...

**Comment by monkpit** [comment]
I installed this theme almost immediately after installing Atom, so I had not yet used the Project Folders feature of Atom. I believe my performance issue is related to having a Project Folder that is a part of a Git Repo. See [this Atom issue](https://github.com/atom/atom/issues/9544) for details.

**Comment by cuiweiqiang** [comment]
@monkpit I installed this theme too, but it switch tabs very fast, even though I used the Project Folders feature of Atom.
Is  computer's configuration does affect in Atom?
My mac is 15's Mid, PC is i7-4770k 16g ram.

**Comment by silvestreh** [comment]
I'm closing this issue as it doesn't seem to be related to `atom-material-ui`. Feel free to reopen it if you have steps to reproduce and the theme is to blame.

**Comment by Obversity** [comment]
@silvestreh I installed this theme and immediately noticed that find-and-replace took slightly longer to open, and significantly longer to close. Similarly, fuzzy-finder and the command palette are slightly slower to load than in the built-in themes. 

This is in a (reasonably large) Rails project. I've tested it in a (significantly smaller) Django project and it seems pretty snappy, with no hint of the above problems. I've also tested it on in a similarly large Ember project and it was just fine. In a fresh rails project it only seems slightly slower than the built-in themes. 

I don't know if those are useful observations. I'd love to use this theme, but just can't justify it with the lag.
